cmd/azure/azure.go

```diff
@@ -187,23 +79,47 @@ func run() error {
 		return err
 	}
 
-	ec, err := newEnvConfig()
+	//action, err := rootCmd.Flags().GetString("action")
+	//if err != nil {
+	//	return err
+	//}
```

Comment:
Since this is commented out, looks like we're not actually using the action flag anywhere yet?

Reply:
yes, preparation for delete logic. But create logic broken due to https://coreos.slack.com/archives/CHFF1LCUR/p1560272128015500